system/alternatives.py

```diff
@@ -123,10 +142,17 @@ def main():
                 if not link:
                     module.fail_json(msg="Needed to install the alternative, but unable to do so as we are missing the link")
 
-                module.run_command(
-                    [UPDATE_ALTERNATIVES, '--install', link, name, path, str(DEFAULT_LINK_PRIORITY)],
-                    check_rc=True
-                )
+                if need_to_force:
```

Comment:
i actually did something like this originally... i'll come up with a new pull request when i have a chance...